### ISSUE TITLE
ReferenceError when assert interval timer of an class instance

### ISSUE THREAD

**Issue by gxcsoccer** [power-assert-js/power-assert#79]
`test/a.js`

```js
 class A {
  constructor() {
    this.timer = setInterval(() => {}, 1000);
  }
}
const a = new A();
assert(a.timer);
```

`index.js`

```js
'use strict';

require('intelli-espower-loader');
require('./test/a');
```

When I run node `index.js`, catch an error.

```bash
test/a.js:11
assert(a.timer);
       ^
ReferenceError: _rec1 is not defined
    at Object.<anonymous> (test/a.js:11:8)
    at Module._compile (module.js:570:32)
    at Object.extensions..js (/Users/gaoxiaochen/projj/github.com/node-modules/tcp-base/node_modules/.1.2.0@espower-loader/index.js:46:25)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/gaoxiaochen/projj/github.com/node-modules/tcp-base/index.js:2:1)
    at Module._compile (module.js:570:32)
```

**Comment by twada** [comment]
@gxcsoccer Thank you for reporting and creating minimal reproduction case. 
This is a same issue as #78 and [I'm fixing it now](https://github.com/power-assert-js/espower/pull/36). So please stay tuned.

**Comment by twada** [comment]
Just released [espower v2.0.3](https://github.com/power-assert-js/espower/releases/tag/v2.0.3) as a bugfix release for this issue. Would you reinstall intelli-espower-loader and give a try again?

**Comment by twada** [comment]
I close this issue since this is the same issue as #78 and it has been fixed. Please feel free to reopen this issue if you have any problems.
